### PR TITLE
Create list spec WIP

### PR DESCRIPTION
**Pull request by margdaly**
__x__ Wrote Tests ____ Implemented ____ Reviewed

## Neccesary checkmarks:

    [] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [x] New feature
    [] Bug Fix
    [] Refactor

## Description of Change:

    description: Added tests for createList
    
## Requested feedback: 
    
    I'd like feedback on... Testing if a list will be created with invalid trip type 
    will you take a look at the commented out code around invalid_tripType? Lines 65-80 & 158-81
    - When given "tripType"=>"Invalid" it does create a list
    1) we can let the creator of the list choose custom trip type 
or
    2) I think we can set variables for graphql to make tripType only allow our 6 categories 
    
    This is just an initial setup trying out testing - we can completely change or refactor it to have spec/fixtures or the queries in the rails helper if preferred! 

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [] My code has no unused/commented out code
    [x] My code has no binding.pry's
    [] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: spec/graphql/mutations/lists/create_list_spec.rb [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+            post '/graphql', params: { query: query_invalid_name }
+            data = JSON.parse(response.body)['data']['createList']
+
+            expect(data['list']).to be_nil
+            expect(data['errors']).to eq(["Name can't be blank"])
+          end
+        end
+
+        # context 'with invalid tripType' do
+        #   it 'does not create a list' do
```
Comment (kohyoungheon) [inline spec/graphql/mutations/lists/create_list_spec.rb:66]: I see that there is a commented out test here. We can try to tackle how to handle invalid trip types together.

File: spec/graphql/mutations/lists/create_list_spec.rb [hunk 2]
```diff
@@ -0,0 +1,184 @@
+require 'rails_helper'
+
+module Mutations
+  module Lists
+    RSpec.describe CreateList, type: :request do
+      describe '.resolve' do
```
Comment (kohyoungheon) [inline spec/graphql/mutations/lists/create_list_spec.rb:6]: The tests look good! Thanks for being on top of it!

**Review by kohyoungheon — APPROVED** [review]
Good job with the tests. Lets tackle the couple edge cases together and see how we should handle them.
